### ISSUE TITLE
Enable optional chaining in babylon parser

### ISSUE THREAD

**Issue by dozortsev** [KnisterPeter/react-to-typescript-definitions#1123]
Hey

I try to use this great library for my library of components and in some of them we use [optional chaining feature](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) and react-to-typescript-definitions fails to generate types for them.

I checked code and found that to fix the issue we just need add `'optionalChaining',` here: https://github.com/KnisterPeter/react-to-typescript-definitions/blob/master/src/index.ts#L134

<details>
  <summary>I would love to submit PR but I can't push, don't have permissions.</summary>
  
```
➜  react-to-typescript-definitions git:(allow-optionalChaining) git push origin allow-optionalChaining  
ERROR: Permission to KnisterPeter/react-to-typescript-definitions.git denied to dozortsev.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```
</details>

@KnisterPeter how could we handle that? 🙏

**Comment by dozortsev** [comment]
I actually was able to create a PR by editing file via Github 😊  **https://github.com/KnisterPeter/react-to-typescript-definitions/pull/1124**

**Comment by KnisterPeter** [comment]
This is released in 3.0.0
